packages/kit/test/utils.js

```diff
@@ -111,20 +111,25 @@ export const test = base.extend({
 	in_view: async ({ page }, use) => {
 		/** @param {string} selector */
 		async function in_view(selector) {
-			// @ts-expect-error
-			return page.$eval(selector, async (element) => {
-				const { top, bottom } = element.getBoundingClientRect();
-
-				if (top > window.innerHeight || bottom < 0) {
-					// slightly more useful feedback than true/false
-					return {
-						top,
-						bottom
-					};
-				}
+			const isVisible = await page.isVisible(selector);
```

Comment:
I might reverse the condition to indent less code:
```
if (visible) return true;
return page.$eval(...
```

Reply:
Done!